arrow-core-test/src/main/kotlin/arrow/test/laws/MonadLaws.kt

```diff
@@ -33,8 +32,6 @@ object MonadLaws {
       listOf(
         Law("Monad Laws: left identity") { M.leftIdentity(G, EQ) },
         Law("Monad Laws: right identity") { M.rightIdentity(G, EQ) },
-        Law("Monad Laws: kleisli left identity") { M.kleisliLeftIdentity(G, EQ) },
-        Law("Monad Laws: kleisli right identity") { M.kleisliRightIdentity(G, EQ) },
```

Comment:
These laws are alternatives to the left and right identity laws. They exist because they show the similarity between monoid laws but are itself not useful when the other two identity laws are in use